presto-main/src/main/java/com/facebook/presto/sql/planner/PlanPrinter.java

```diff
@@ -544,6 +562,25 @@ private void printDistributions(int indent, PlanNodeStats nodeStats)
         return ImmutableMap.of();
     }
 
+    private void printWindowNodeStats(int indent, WindowNodeStats stats)
+    {
+        output.append(indentString(indent));
+        output.append(format("Active Drivers: [ %d / %d ]", stats.getActiveDrivers(), stats.getTotalDrivers()));
```

Comment:
When discussing with @cberner about `EXPLAIN ANALYZE` we concluded that showing the number of drivers to the user doesn't make sense. Driver is internal concept to Presto and one has to understand internals to understand this data. See discussion: https://github.com/prestodb/presto/pull/5499#discussion_r90699145